v2/devices/Spacewar.yml

```diff
@@ -0,0 +1,188 @@
+name: "Nothing Phone 1"
+codename: "Spacewar"
+formfactor: "phone"
+aliases: ["Spacewar"]
```

Comment:
You don't need an alias for the same codename already provided above. If the device can appear under multiple (for example a different one in fastboot than adb) you can put them here, otherwise remove it.

Reply:
Fixed